src/AppBundle/Controller/Report/AccountController.php

```diff
@@ -31,7 +31,10 @@ public function addAccountAction(Request $request, $reportId)
 
         $this->fillAccountData($account, $data);
 
+        $report->updateSectionStatus(Report::SECTION_BANK_ACCOUNTS);
```

Comment:
Optional: have a sectionId as a private property in each controller:
private $sectionId = Report::SECTION_BANK_ACCOUNTS;
then your update becomes:
$report->updateSectionStatus($this->sectionId);

Reply:
that was done